droidlet/lowlevel/hello_robot/remote/remote_hello_robot.py

```diff
@@ -58,11 +59,13 @@ def __init__(self, ip):
         if not self._robot.is_calibrated():
             self._robot.home()
         self._robot.stow()
+        self._lidar = Lidar()
```

Comment:
why not have `self._lidar` be part of `remote_hello_realsense` instead of this one? It more aligns as "preceptual sensor" and that's the only service that uses it right no anyways.
We can rename `remote_hello_realsense` to `remote_hello_perception` if needed